tools/convert_datasets/a2d2.py

```diff
@@ -0,0 +1,389 @@
+    (200, 0, 0):      13,  # Car 2 --> Car
+    (150, 0, 0):      13,  # Car 3 --> Car
+    (128, 0, 0):      13,  # Car 4 --> Car
+    (182, 89, 6):     18,  # Bicycle 1 --> Bicycle
+    (150, 50, 4):     18,  # Bicycle 2 --> Bicycle
+    (90, 30, 1):      18,  # Bicycle 3 --> Bicycle
+    (90, 30, 30):     18,  # Bicycle 4 --> Bicycle
+    (204, 153, 255):  11,  # Pedestrian 1 --> Person
+    (189, 73, 155):   11,  # Pedestrian 2 --> Person
+    (239, 89, 191):   11,  # Pedestrian 3 --> Person
+    (255, 128, 0):    14,  # Truck 1 --> Truck
+    (200, 128, 0):    14,  # Truck 2 --> Truck
+    (150, 128, 0):    14,  # Truck 3 --> Truck
+    (0, 0, 100):      14,  # Tractor --> Truck ?
+    (0, 255, 0):      17,  # Small vehicles 1 --> Motorcycle ?
+    (0, 200, 0):      17,  # Small vehicles 2 --> Motorcycle ?
+    (0, 150, 0):      17,  # Small vehicles 3 --> Motorcycle ?
+    (0, 128, 255):     6,  # Traffic signal 1 --> Traffic light
+    (30, 28, 158):     6,  # Traffic signal 2 --> Traffic light
+    (60, 28, 100):     6,  # Traffic signal 3 --> Traffic light
+    (0, 255, 255):     7,  # Traffic sign 1 --> Traffic sign
+    (30, 220, 220):    7,  # Traffic sign 2 --> Traffic sign
+    (60, 157, 199):    7,  # Traffic sign 3 --> Traffic sign
+    (255, 255, 0):    13,  # Utility vehicle 1 --> Car ?
+    (255, 255, 200):  13,  # Utility vehicle 2 --> Car ?
+    (233, 100, 0):     4,  # Sidebars --> Fence
+    (110, 110, 0):     0,  # Speed bumper --> Road
+    (128, 128, 0):     1,  # Curbstone --> Sidewalk
+    (255, 193, 37):    0,  # Solid line --> Road
+    (64, 0, 64):     255,  # Irrelevant signs --> Ignore
+    (185, 122, 87):    4,  # Road blocks --> Fence
+    (139, 99, 108):    0,  # Non-drivable street --> Road
+    (210, 50, 115):    0,  # Zebra crossing --> Road
+    (255, 0, 128):   255,  # Obstacles / trash --> Ignore ?
+    (255, 246, 143):   5,  # Poles --> Poles
+    (150, 0, 150):     0,  # RD restricted area --> Road ?
+    (204, 255, 153): 255,  # Animals --> Ignore ?
+    (238, 162, 173):   0,  # Grid structure --> Road ???
+    (33, 44, 177):     6,  # Signal corpus --> Traffic light ?
+    (180, 50, 180):    0,  # Drivable cobblestone --> Road
+    (255, 70, 185):  255,  # Electronic traffic --> Ignore ???
+    (238, 233, 191):   0,  # Slow drive area --> Road
+    (147, 253, 194):   8,  # Nature object --> Vegetation
+    (150, 150, 200):   0,  # Parking area --> Road
+    (180, 150, 200):   1,  # Sidewalk --> Sidewalk
+    (72, 209, 204):  255,  # Ego car --> Ignore
+    (200, 125, 210):   0,  # Painted driv. instr. --> Road ???
+    (159, 121, 238):   0,  # Traffic guide obj. --> Road ???
+    (128, 0, 255):     0,  # Dashed line --> Road
+    (255, 0, 255):     0,  # RD normal street --> Road
+    (135, 206, 255):  10,  # Sky --> Sky
+    (241, 230, 255):   2,  # Buildings --> Building
+    (96, 69, 143):   255,  # Blurred area --> Ignore ?
+    (53, 46, 82):    255,  # Rain dirt --> Ignore ?
+}
```

Comment:
There are 14 categories in 'SEG_COLOR_DICT_CITYSCAPES' which are subclasses of CITYSCAPES, so we can use [subcalss train](https://github.com/open-mmlab/mmsegmentation/blob/5a76a715f10afc58d47522a99bffc7c31da2642f/mmseg/datasets/custom.py#L61) in dataset and model configs.

Reply:
Sorry, could you please elaborate on what you think should be modified?